From 62c7cb5839b24329445716b23dd98756b09757d3 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Sun, 29 Mar 2009 22:33:57 +0200
Subject: [PATCH] m68k: Hydra Ethernet - correct printing of resource_size_t

| drivers/net/hydra.c:179: warning: format '%08lx' expects type 'long unsigned int', but argument 3 has type 'resource_size_t'

resource_size_t changed from `unsigned long' to `u32' on 32-bit. Print
the whole resource to make it future-proof.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/net/hydra.c |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/net/hydra.c b/drivers/net/hydra.c
index 8a6d070..18257e6 100644
--- a/drivers/net/hydra.c
+++ b/drivers/net/hydra.c
@@ -174,9 +174,8 @@ static int __devinit hydra_init(struct zorro_dev *z)
 
     zorro_set_drvdata(z, dev);
 
-    printk(KERN_INFO "%s: Hydra at 0x%08llx, address "
-	   "%pM (hydra.c " HYDRA_VERSION ")\n",
-	   dev->name, (unsigned long long)z->resource.start, dev->dev_addr);
+    pr_info("%s: Hydra at %pR, address %pM (hydra.c " HYDRA_VERSION ")\n",
+	    dev->name, &z->resource, dev->dev_addr);
 
     return 0;
 }
-- 
1.5.6.5

